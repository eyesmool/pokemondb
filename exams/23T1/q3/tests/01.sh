psql bank -c 'select * from q3 order by branch;'
