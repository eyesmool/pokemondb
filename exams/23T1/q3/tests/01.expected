  branch   
-----------
 Blacktown
(1 row)
